torch/_dynamo/eval_frame.py

```diff
@@ -1705,7 +1705,7 @@ def export(
     _log_export_usage: bool = True,
     constraints: Optional[list[Constraint]] = None,
     **extra_kwargs: Any,
-) -> Callable[[tuple[Any, Any]], ExportResult]:
+) -> Callable[..., ExportResult]:
```

Comment:
tuple[Any, Any] isn't quite right since truly this is args and kwargs; for now we can use the less restrictive `...` here

Reply:
Can't you do tuple[...] ?